.idea/google-java-format.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Code repositories should only contain:

1. Source code
2. Any configuration files needed to build and run the app(s) in the repo
3. Any documentation

In your case, this repo should just contain:

1. ```src```
2. ```movies.txt```
3. ```README.md```

All the files found in the ```.idea/``` directory are autogenerated by IntelliJ for _your particular set up_. Other developers may not use IntelliJ, and even if they do, they'll probably set it up differently. Therefore we should not check them in. To get git to ignore these files, do the following:

1. Close down IntelliJ
2. Copy (_not_ move) ```.idea/``` and ```GuessTheMovie.iml``` to a safe directory outside this repo
3. Now we can get git to remove them from the repo with ```git rm GuessTheMovie.iml``` and ```git rm -r .idea/```
4. Now commit these deletions with ```git add .```, ```git commit -m "Removed IDE files"```
5. Now, when you open IntelliJ again (*not yet!*) those files may be created and checked in again. To avoid this, we tell git to ignore them. Create a file called ```.gitignore``` in the root of your project and on separate lines, put these two entries ```idea/*``` and ```*.iml```
6. Now commit this file, ```git add .```, ```git commit -m "Git now ignores IntelliJ files"```
7. Do this on _this_ branch you created ```issue-1``` and push them. This PR will change accordingly.
8. Now that you've done this, you can copy the ```.idea/``` and ```GuessTheMovie.iml``` files back into your project and git will happily ignore them.
9. Open IntelliJ again.

So, your ```.gitignore``` file should look like this:

```
.idea/*
*.iml
```

Reply:
Next step, do this exact same thing for your ```out``` directory!